tests/ops/qubit/test_attributes.py

```diff
@@ -151,6 +151,15 @@ class TestSupportsBroadcasting:
     """Test that all operations in the ``supports_broadcasting`` attribute
     actually support broadcasting."""
 
+    @pytest.fixture(scope="function", autouse=False)
```

Comment:
Why this change is needed in a deprecation PR?

Reply:
I personally would say we can just skip `QubitStateVector` attribute testing for its deprecation cycle.